docs/api/IoT.js-API-DNS.md

```diff
@@ -16,30 +16,30 @@ implemented on top of the underlying operation system facilities. Please see
 [Implementation considerations section](#implementation-considerations).
 
 The following properties are flags which can be passed as hints for the
-[`dns.lookup()`](#dnslookuphostname-options-callback) method.
+[dns.lookup()](#dnslookuphostname-options-callback) method.
```

Comment:
We are keeping the aphostrophe in the references, but not in the section headers.